pkgs/development/python-modules/bytecode/default.nix

```diff
@@ -2,23 +2,31 @@
 , buildPythonPackage
 , fetchFromGitHub
 , pythonOlder
+, pytestCheckHook
 , aenum
 }:
 
 buildPythonPackage rec {
   pname = "bytecode";
-  version = "0.11.0";
+  version = "0.12.0";
+  disabled = pythonOlder "3.6";
 
   src = fetchFromGitHub {
     owner = "vstinner";
     repo = pname;
     rev = version;
-    sha256 = "097k83zr0z71pha7bafzhs4ink174wk9ls2883bic274rihsnc5r";
+    sha256 = "sha256-CEfDoJ+JlnxLLVnSxv7bEN891tmwG9zAvtT8GNvp8JU=";
   };
 
-  disabled = pythonOlder "3.5";
+  propagatedBuildInputs = lib.optionals (pythonOlder "3.6") [
```

Comment:
since we have `disabled = pythonOlder "3.6"`, do we still need this conditional here?

Reply:
No, we don't.